src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1714,3 +1714,18 @@ Executing_command_\"%0\"...=
 Error_occured_while_executing_the_command_\"%0\".=
 
 Reformat_ISSN=
+
+Add_protected_term_file=
+Add_term_file=
+Are_you_sure_you_want_to_remove_the_term_list?=
+Countries_and_territories_in_English=
+Electrical_engineering_terms=
+Enabled=
+Internal_list=
+Manage_term_files=
+Months_and_weekdays_in_English=
+No=
```

Comment:
Aren't there any other translated strings for "yes" or "no"? Could we be more specific somehow? See our discussion on the database-changed-what-do-you-want-to-do-dialog.